scalableminds/webknossos
# Add URLs for dashboard tabs

This PR adds paths to directly access a tab of the dashboard.

### URL of deployed dev instance (used for testing):
- [https://addurlfordashboard.webknossos.xyz](https://addurlfordashboard.webknossos.xyz)

### Steps to test:
- `.../dashboard/gallery` should open the `dataset gallery`
- `.../dashboard/datasets` should open the `dataset-list view`
- `.../dashboard/tasks` should open the `tasks view`
- `.../dashboard/annotations` should open the `explorative annotations view`

### Issues:
- fixes #3134

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
